File: 1_Java_Ecosystem/Spring_Projects/springDIDemo/src/main/java/com/lyuke/springdidemo/Student/Student.java
package com.lyuke.springdidemo.Student;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class Student {

    private String name;
    private String denger;
    private String email;
    private Integer age;


}

File: 6_Py/Jul-11/Student.py

class Student:

    ClassName = "高三八班"

    def __init__(self,name,age):
        self.name = name
        self.age = age

    def __str__(self):
        return f"{self.name} and {self.age}"

    def run(self):
        print(f"你好，我是学生 {self.name}")

    def eat(self):
        print(f"{self.name}:正在吃饭")